st2exporter/st2exporter/worker.py

```diff
@@ -13,118 +13,65 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import Queue
-
 import eventlet
+
 from kombu import Connection
-from oslo_config import cfg
 
 from st2common import log as logging
-from st2common.constants.action import (LIVEACTION_STATUS_SUCCEEDED, LIVEACTION_STATUS_FAILED,
-                                        LIVEACTION_STATUS_CANCELED)
-from st2common.models.api.execution import ActionExecutionAPI
-from st2common.models.db.execution import ActionExecutionDB
-from st2common.persistence.execution import ActionExecution
-from st2common.persistence.marker import DumperMarker
-from st2common.transport import consumers, execution, publishers
+from st2common.transport import execution, publishers
 from st2common.transport import utils as transport_utils
-from st2common.util import isotime
-from st2exporter.exporter.dumper import Dumper
+from st2exporter.consumers.execution import ExecutionsExporter
 
-__all__ = [
-    'ExecutionsExporter'
-]
-
-COMPLETION_STATUSES = [LIVEACTION_STATUS_SUCCEEDED, LIVEACTION_STATUS_FAILED,
-                       LIVEACTION_STATUS_CANCELED]
-LOG = logging.getLogger(__name__)
 
 EXPORTER_WORK_Q = execution.get_queue(
     'st2.exporter.work', routing_key=publishers.UPDATE_RK)
+LOG = logging.getLogger(__name__)
+
 
+class Exporter(object):
+    def __init__(self):
+        self._model_exporters = []
```

Comment:
First I was worried we will potentially have one worker (process) for each model which we want to export, but this looks better :)